pycytominer/cyto_utils/util.py

```diff
@@ -17,10 +18,33 @@
 
 
 def get_default_compartments():
+    """Returns default compartments.
+
+    Returns
+    -------
+    list of str
+        Default compartments.
+
+    """
+
     return ["cells", "cytoplasm", "nuclei"]
 
 
 def check_compartments(compartments):
+    """Checks if the input compartments are non canonical compartments.
```

Comment:
```suggestion
    """Checks if the input compartments are noncanonical compartments.
```

Reply:
I have some thoughts. I will move the discussion to #130